Dockerfile

```diff
@@ -12,8 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:stretch-slim
+FROM golang:1.12-stretch AS build
+WORKDIR /go/src/github.com/vmware-tanzu/velero-plugin-for-aws
+# copy vendor in separately so the layer can be cached if the contents don't change
+COPY vendor vendor
+COPY velero-plugin-for-aws velero-plugin-for-aws
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o /go/bin/velero-plugin-for-aws ./velero-plugin-for-aws
+
+
+FROM ubuntu:bionic AS run
```

Comment:
this step doesn't actually need a name, right?

Reply:
correct, I can drop it - left it there to be "self-documenting" but it doesn't matter